src/main/java/prefs/GlobalConfig.java

```diff
@@ -28,6 +28,7 @@
     private String lastViewedRepository = "";
     private String lastLoginUsername = "";
     private byte[] lastLoginPassword = new byte[0];
+    private String lastOpenBoard = "";
```

Comment:
We should use `Optional<String>` here instead. Expresses the semantics of this field more clearly.

Also, the empty string is a valid board name... Probably shouldn't be, but its use here as an empty value could cause problems.